src/test/rating_test.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  iit,
+  it,
+  ddescribe,
+  describe,
+  expect,
+  inject,
+  injectAsync,
+  TestComponentBuilder,
+  beforeEachProviders
+} from 'angular2/testing';
+
+import {Component} from 'angular2/angular2';
+
+import {NgbRating} from '../components/rating';
+
+describe('ngb-rating', () => {
+  it('should show 10 stars by default', injectAsync([TestComponentBuilder], (tcb) => {
+       return tcb.createAsync(NgbRating).then((fixture) => {
+         fixture.detectChanges();
+
+         const compiled = fixture.debugElement.nativeElement;
+
+         const stars = compiled.querySelectorAll('i');
+         expect(stars.length).toBe(10);
+       });
+     }));
+
+  it('should change the num of stars with `max`', injectAsync([TestComponentBuilder], (tcb) => {
+       const html = '<ngb-rating max="3"></ngb-rating>';
+
+       return tcb.overrideTemplate(TestComponent, html)
+           .createAsync(TestComponent)
+           .then((fixture) => {
```

Comment:
I'm fine with leaving that alone then - more of a code style comment than anything else.